tests/util/skvbc_history_tracker.py

```diff
@@ -901,4 +901,91 @@ def writeset(self, max_size):
                     await self.send_tracked_read(client, max_size)
             except:
                 pass
-            await trio.sleep(.1)
\ No newline at end of file
+            await trio.sleep(.1)
+
+    async def write_and_track_known_kv(self, kv, client):
```

Comment:
I really like those tracked read/write utilities, but there's some duplication already with the skvbc.py module. 

I suggest to keep it like this for now. Later on we might consider injecting an optional SkvbcTracker into the SKVBC Protocol and extracting the common logic.

I'll add a backlog entry for this.

Reply:
Yes I know, I did kind of identical functions but with using the tracking capability and still preserved the existing functions because I didn't want to delete without getting permission.